scripts/tests.sh

```diff
@@ -30,6 +30,20 @@ set -e
 
 REPO_ROOT="$(dirname "$0")"/..
 
+if [ "$1" -e --junit_report ]
+then
+    if [ -z "$2" ]
+    then
+        echo "Usage: $0 [--junit_report <report_directory>]"
+        exit 1
+    fi
+    testargs_no_opt="--logger=JUNIT,test_suite,$2/no_opt.xml"
+    testargs_opt="--logger=JUNIT,test_suite,$2/opt.xml"
```

Comment:
What is `--logger==JUNIT`? Is it useful with circleci?

Reply:
Yes, it creates an XML file with test results that is parsed by circle. It then displays which exact test failed at the top of the build page.